docs/index.rst

```diff
@@ -9,14 +9,49 @@ Yellowbrick: Machine Learning Visualization
+Yellowbrick helps you make better models through visualization. 
+________________________________________________________________
 
-If you're new to Yellowbrick, checkout the :doc:`quickstart` or skip ahead to the :doc:`tutorial`. Yellowbrick is a rich library with many Visualizers being added on a regular basis. For details on specific Visualizers and extended usage head over to the :doc:`api/index`. Interested in contributing to Yellowbrick? Checkout the :ref:`contributing guide <contributing>` . If you've signed up to do user testing, head over to the :doc:`evaluation` (and thank you!).
+Yellowbrick extends the Scikit-Learn API to make model selection and hyperparameter tuning easier. Under the hood, it's using Matplotlib. 
+
+Recommended Learning Path
+-------------------------
+
+1. Check out the :doc:`quickstart` 
+
+2. Try the the :doc:`tutorial`. 
+
+3. Use Yellowbrick in your project. 
+
+4. For details on specific Visualizers and extended usage head over to the :doc:`api/index`. 
+
+5. Follow us on Twitter `@scikit_yb <https://twitter.com/scikit_yb>`_. You'll hear about new visualizers as they're added. 
+
+Contributing
+------------
+
+Interested in contributing to Yellowbrick? Yellowbrick is a welcoming, inclusive project and we would love to have you. We follow the `Python Software Foundation Code of Conduct <http://www.python.org/psf/codeofconduct/>`_. 
+
+No matter your level of technical skill, you can be helpful. We appreciate bug reports, user testing, feature requests, bug fixes, product enhancements, and documentation improvements. 
+
+Check out the :doc:`contributing/index` guide!
+
+If you've signed up to do user testing, head over to the :doc:`evaluation`.
+
+Please consider joining the `Google Groups Listserv <https://groups.google.com/forum/#!forum/yellowbrick>`_ listserve so you can respond to questions.
+
+Thank you for your contributions!
+
+Concepts & API
+--------------
 
 Visualizers
 -----------
+We strive to maintain a brain-friendly API. Our API is similar to Scikit-Learn's API.
 
-Visualizers are estimators (objects that learn from data) whose primary objective is to create visualizations that allow insight into the model selection process. In Scikit-Learn terms, they can be similar to transformers when visualizing the data space or wrap an model estimator similar to how the "ModelCV" (e.g. `RidgeCV <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.RidgeCV.html>`_, `LassoCV <http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LassoCV.html>`_) methods work. The primary goal of Yellowbrick is to create a sensical API similar to Scikit-Learn. Some of our most popular visualizers include:
```

Comment:
We like these 2 sentences: "Visualizers are estimators..." and "The primary goal of Yellowbrick..." Perhaps we could them back in?  Thanks!

Reply:
Sure.